bench/readv.ml

```diff
@@ -0,0 +1,60 @@
+(* This benchmark uses [Uring.readv] to read from /dev/zero in a loop.
+   This is intended to stress the sketch buffer. *)
+
+let buffer_size = 100   (* Use a small buffer to stress the system more *)
```

Comment:
As a note, the performance drop relevant to buffer size is consistent to what I observed in my old `catv`, 128k was the sweet spot for me, it's also what GNU's cat was using.